pages/box_buster.vue

```diff
@@ -153,6 +156,8 @@ export default {
         // Requirements were that we should allow plate lookups
         this.findPlates({ barcodes: [this.barcode] })
       }
+      this.barcodes_scanned.push(this.barcode)
+      this.barcode = ''
```

Comment:
Maybe these would be better after line 171, so only removing the barcodes if the response is successful?

Reply:
Have you thought about using a computed value with v-model?